src/vos/vos_internal.h

```diff
@@ -417,7 +417,8 @@ vos_feats_agg_time_get(uint64_t feats, daos_epoch_t *epoch)
 
 	if (feats & VOS_TF_AGG_HLC) {
 		*epoch = ((feats & VOS_TF_AGG_TIME_MASK) << VOS_TF_AGG_RSHIFT);
-		*epoch += 0xffffffffULL;
+		if (*epoch != 0)
```

Comment:
can epoch be zero when VOS_TF_AGG_HLC is present?

Reply:
Yes.  If the tree has only a single update, it will be 0

Reply:
Niu correctly pointed out that VOS_TF_AGG_HLC will not be set in this case.